From 30a59cbbfb46a5a47e13c6e5bd798665a30b2a2d Mon Sep 17 00:00:00 2001
From: Aikar <aikar@aikar.co>
Date: Wed, 26 Aug 2015 22:50:43 -0400
Subject: [PATCH] Timings v2: Add Trove to Bukkit

Let Plugin Developers (And Timings) use Trove classes, and mark guava as provided vs compile
---
 pom.xml | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/pom.xml b/pom.xml
index 5244305..2ec1700 100644
--- a/pom.xml
+++ b/pom.xml
@@ -56,6 +56,13 @@
 
     <dependencies>
         <dependency>
+            <groupId>net.sf.trove4j</groupId>
+            <artifactId>trove4j</artifactId>
+            <version>3.0.3</version>
+            <!-- Trove Provided by CraftBukkit -->
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
             <groupId>commons-lang</groupId>
             <artifactId>commons-lang</artifactId>
             <version>2.6</version>
@@ -73,7 +80,7 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
             <version>17.0</version>
-            <scope>compile</scope>
+            <scope>provided</scope>
         </dependency>
         <!-- bundled with Minecraft, should be kept in sync -->
         <dependency>
-- 
1.9.1

